netbox/templates/extras/script_result.html

```diff
@@ -41,8 +41,37 @@
     {# Object list tab #}
     <div class="tab-pane show active" id="results" role="tabpanel" aria-labelledby="results-tab">
 
+  <p>
+    {% if job.started %}
+      {% trans "Started" %}: <strong>{{ job.started|isodatetime }}</strong>
+    {% elif job.scheduled %}
+      {% trans "Scheduled for" %}: <strong>{{ job.scheduled|isodatetime }}</strong>
+    {% else %}
+      {% trans "Created" %}: <strong>{{ job.created|isodatetime }}</strong>
+    {% endif %}
+    {% if job.completed %}
+      {% trans "Duration" %}: <strong>{{ job.duration }}</strong>
+    {% endif %}
+    <span id="pending-result-label">{% badge job.get_status_display job.get_status_color %}</span>
+  </p>
```

Comment:
These lines need to stay in the HTMX template so that the script's status is updated on each refresh. (Otherwise, it will still read "running" when the script has completed.)